samples/bluetooth/peripheral_hids_keyboard/boards/nrf54lv10dk_nrf54lv10a_cpuapp.conf

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright (c) 2025 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+CONFIG_MAIN_STACK_SIZE=3584
+
+# Workaround for issue NCSDK-33313
+
+# Disable Cracken for all but RNG, and switch to software crypto.
+
+CONFIG_PSA_CRYPTO_DRIVER_OBERON=y
+CONFIG_PSA_CRYPTO_DRIVER_CRACEN=y
+
+CONFIG_PSA_USE_CRACEN_CIPHER_DRIVER=n
+CONFIG_PSA_USE_CRACEN_AEAD_DRIVER=n
+CONFIG_PSA_USE_CRACEN_HASH_DRIVER=n
+CONFIG_PSA_USE_CRACEN_MAC_DRIVER=n
+CONFIG_PSA_USE_CRACEN_KEY_AGREEMENT_DRIVER=n
+CONFIG_PSA_USE_CRACEN_ASYMMETRIC_DRIVER=n
+CONFIG_PSA_USE_CRACEN_KEY_MANAGEMENT_DRIVER=n
+CONFIG_PSA_USE_CRACEN_KEY_DERIVATION_DRIVER=n
+CONFIG_PSA_USE_CRACEN_PAKE_DRIVER=n
+CONFIG_PSA_USE_CRACEN_ASYMMETRIC_SIGNATURE_DRIVER=n
+
+# END of Workaround for issue NCSDK-33313
+
+CONFIG_NFC_OOB_PAIRING=n
```

Comment:
not needed if the nfct DTS node is disabled or deleted

In this case, the `CONFIG_HAS_HW_NRF_NFCT` should be disabled and it is a dependency of the `CONFIG_NFC_OOB_PAIRING`